engine/apps/labels/utils.py

```diff
@@ -42,19 +42,28 @@ def get_associating_label_model(obj_model_name: str) -> typing.Type["AssociatedL
     return label_model
 
 
-def is_labels_feature_enabled(organization) -> bool:
-    """
-    Checks if labels feature enabled for all organizations (FEATURE_LABELS_ENABLED_FOR_ALL).
-    If not, checks if current organization's grafana org_id is in the list of organizations labels feature enabled for
-    (FEATURE_LABELS_ENABLED_FOR_GRAFANA_ORGS)
-    """
-    logger.info(
-        "is_labels_feature_enabled: "
-        f"FEATURE_LABELS_ENABLED_FOR_ALL={settings.FEATURE_LABELS_ENABLED_FOR_ALL}, "
-        f"organization in FEATURE_LABELS_ENABLED_FOR_GRAFANA_ORGS="
-        f"{organization.id in settings.FEATURE_LABELS_ENABLED_FOR_GRAFANA_ORGS}, "
-        f"organization={organization.id}"
+def is_labels_feature_enabled(organization: "Organization") -> bool:
+    return (
+        settings.FEATURE_LABELS_ENABLED_FOR_ALL
+        or organization.org_id in settings.FEATURE_LABELS_ENABLED_FOR_GRAFANA_ORGS  # Grafana org ID, not OnCall org ID
     )
-    if not settings.FEATURE_LABELS_ENABLED_FOR_ALL:
-        return organization.org_id in settings.FEATURE_LABELS_ENABLED_FOR_GRAFANA_ORGS
-    return settings.FEATURE_LABELS_ENABLED_FOR_ALL
+
+
+def assign_labels(alert_group: "AlertGroup", alert_receive_channel: "AlertReceiveChannel") -> None:
+    from apps.labels.models import AlertGroupAssociatedLabel
+
+    if not is_labels_feature_enabled(alert_receive_channel.organization):
+        return
+
+    # inherit all labels from the integration
+    # FIXME: this is a temporary solution before we have a UI for configuring inherited labels
+    alert_group_labels = [
+        AlertGroupAssociatedLabel(
+            alert_group=alert_group,
+            organization=alert_receive_channel.organization,
+            key_name=label.key.name,
+            value_name=label.value.name,
+        )
+        for label in alert_receive_channel.labels.all().select_related("key", "value")
+    ]
+    AlertGroupAssociatedLabel.objects.bulk_create(alert_group_labels)
```

Comment:
Passing ALL integration labels down to alert groups for now, so it's easier to create labels for alert groups locally